### ISSUE TITLE
Events displayed in reverse chronological order

### ISSUE THREAD

**Issue by flyaswish** [rashidkpc/kibana2#75]
Noticed events displayed in Kibana are in reverse chronological order, like below (see timestamp in the message field):

Time    @message
09/20 11:30:20  11:30:20,243 DEBUG [Thread-2744]  - record filterd in code by indicator :0
09/20 11:30:20  11:30:20,231 INFO  [Thread-2744]  - found 1 results
09/20 11:30:20  11:30:20,144 DEBUG [Thread-2750]  - record filterd in code by indicator :0
09/20 11:30:20  11:30:20,131 INFO  [Thread-2750]  - found 1 results
09/20 11:30:20  11:30:20,130 INFO  [Thread-2750]  - query is :@custom:ContentID:"3641009"
09/20 11:30:20  11:30:20,130 DEBUG [Thread-2750]  - exluding:

However logstash UI preserves event sequence(see timestamp in the event field):

timestamp   event
2012-09-20T02:08:20.434000Z 
12:08:20,020 DEBUG [Thread-2968]  - maxSearchResults set to :1
2012-09-20T02:08:20.436000Z 
12:08:20,020 DEBUG [Thread-2968]  - exluding:
2012-09-20T02:08:20.438000Z 
12:08:20,020 INFO  [Thread-2968]  - query is :@custom:ContentID:"5715266"
2012-09-20T02:08:20.440000Z 
12:08:20,021 DEBUG [Thread-2973]  - maxSearchResults set to :1
2012-09-20T02:08:20.444000Z 
12:08:20,021 DEBUG [Thread-2973]  - exluding:
2012-09-20T02:08:20.447000Z 
12:08:20,021 INFO  [Thread-2973]  - query is :@custom:ContentID:"5716715"
2012-09-20T02:08:20.449000Z 
12:08:20,021 DEBUG [Thread-2971]  - maxSearchResults set to :1
2012-09-20T02:08:20.451000Z 
12:08:20,021 DEBUG [Thread-2971]  - exluding:
2012-09-20T02:08:20.453000Z 
12:08:20,021 INFO  [Thread-2971]  - query is :@custom:ContentID:"5712968"
2012-09-20T02:08:20.454000Z 
12:08:20,021 DEBUG [Thread-2970]  - maxSearchResults set to :1
2012-09-20T02:08:20.456000Z 
12:08:20,021 DEBUG [Thread-2970]  - exluding:
2012-09-20T02:08:20.458000Z 
12:08:20,022 INFO  [Thread-2970]  - query is :@custom:ContentID:"5712971"
2012-09-20T02:08:20.461000Z 
12:08:20,022 DEBUG [Thread-2969]  - maxSearchResults set to :1
2012-09-20T02:08:20.464000Z 
12:08:20,022 DEBUG [Thread-2974]  - maxSearchResults set to :1
2012-09-20T02:08:20.465000Z 
12:08:20,022 DEBUG [Thread-2969]  - exluding:
2012-09-20T02:08:20.469000Z 
12:08:20,022 DEBUG [Thread-2974]  - exluding:
2012-09-20T02:08:20.470000Z 
12:08:20,022 INFO  [Thread-2969]  - query is :@custom:ContentID:"5713112"
2012-09-20T02:08:20.474000Z 
12:08:20,022 INFO  [Thread-2974]  - query is :@custom:ContentID:"5713019"
2012-09-20T02:08:20.479000Z 
12:08:20,022 INFO  [Thread-2968]  - found 1 results
2012-09-20T02:08:20.482000Z 
12:08:20,023 INFO  [Thread-2973]  - found 1 results
2012-09-20T02:08:20.484000Z 
12:08:20,023 INFO  [Thread-2971]  - found 1 results
2012-09-20T02:08:20.490000Z 
12:08:20,024 INFO  [Thread-2974]  - found 1 results

Is there a config in Kibana to change the display to chronological order?

**Comment by jlivas** [comment]
I would also love a way to flip the steam the other way around!

**Comment by LeonidShamis** [comment]
Hi we also would like an option to display events in a chronological order and not in reverse chronological order, as it is done in Kibana by default. Could it be made a configuration option?
Thank you.

**Comment by rashidkpc** [comment]
Current plan is to make the Time field sortable in the interface vs a config option.

**Comment by LeonidShamis** [comment]
Hi, thank you for the prompt response. Sort by time is also good. I'm not aware of the details and therefore my concern at this stage would be that if the selection (change of Time sort) is not made persistent (which introduces requirement for "state" management of user selections), then it would potentially be reset with each retrieve/filter of different data.
Appreciate your help.

**Comment by rashidkpc** [comment]
An interesting point, that does sound like a case for being able to set the default order.

**Comment by flyasfish** [comment]
I think people who used to with one type of sort will not likely to change it frequently, so make it a configurable item will be better than a change on the UI.

**Comment by dav3860** [comment]
No, it's better to have a sortable column in the UI. Maybe the columns layout could be added to a future "saved searches" feature. To me it makes sense to save a particular search with its user-defined columns layout.
